.github/workflows/chromatic.yml

```diff
@@ -18,7 +18,7 @@ jobs:
         run: npm run build -- --filter=@medplum/react
 
       - name: Publish @medplum/react Chromatic
-        uses: chromaui/action@v10
+        uses: chromaui/action@latest
```

Comment:
Recommended value from Chromatic docs: https://www.chromatic.com/docs/github-actions/

Hoping this will fix broken Chromatic action 🤞

Reply:
Narrator: It did not.

https://github.com/medplum/medplum/actions/runs/7680448959/job/20932503063